test-scripts/master.py

```diff
@@ -260,7 +263,7 @@ def setup_cluster(tconf, hosts, ips, conf, nocon):
             _c.append(str(_p))  # Adding port number for server
             _c.append(str(_p + 1))  # Port number for db
             _svr_name = _h.split('.')[0]    # PBS_SERVER_NAME, but UNUSED?
-            _svrs.append('%s:%d' % (_svr_name, _p)) # probably used for PSI, but UNUSED?
+            _svrs.append('pbs-server-%d:%d' % (_scnt, _p)) # probably used for PSI, but UNUSED?
```

Comment:
sounds like you are not sure about what this is for, so maybe don't touch it?

Reply:
Well, I do. I used your comment. Updated now.